### ISSUE TITLE
Adding more Northstar Icons or being able to inject Custom Icons.

### ISSUE THREAD

**Issue by bxav** [OfficeDev/microsoft-teams-ui-component-library#62]
I'm playing around with the library but I'm quickly limited by the Icon Component. It would be great to have some more other standard icons on it. 

```
  switch (icon) {
    case "Add":
      iconElement = <AddIcon outline />;
      break;
    // Example the calendar Icon  
    case "Calendar":
      iconElement = <CalendarIcon outline />;
      break;
 ```
 
 Or just to be able to inject an icon component in the config of a list or the board. Something like:
 
 ```
 rows: {
      r4: {
        c1: "Whatever",
        actions: {
          edit: { title: "Edit", icon: "Edit" },
          duplicate: { title: "Duplicate", icon: <DuplicateIcon outline /> }
        },
      },
    },
```

**Comment by davezuko** [comment]
100% agreed. I threw together a PR for this: https://github.com/OfficeDev/microsoft-teams-ui-component-library/pull/63

**Comment by thure** [comment]
Apologies for the long delay, but we've arrived at an implementation that doesn't change this library's API that makes all Fluent UI icons available: #69.
